tests/test_rain.py

```diff
@@ -64,6 +64,9 @@ def test_rain_expected(requests_mock):
     rain = client.get_rain(latitude=48.8075, longitude=2.24028)
     date_rain = rain.next_rain_date_locale()
     assert str(date_rain) == "2020-05-20 19:50:00+02:00"
+    assert rain.timestamp_to_locale_time(
+        rain.forecast[3]["dt"] == "2020-05-20 19:50:00+02:00"
+    )
```

Comment:
ça serait pas 

```suggestion
    assert str(rain.timestamp_to_locale_time(rain.forecast[3]["dt"])) == "2020-05-20 19:50:00+02:00"
```

plutôt ? (ou un truc du style)

Reply:
Effectivement c'est mieux. Etrangement le test passe.